counter/src/main/java/edu/nextstep/camp/counter/CounterViewModel.kt

```diff
@@ -0,0 +1,37 @@
+package edu.nextstep.camp.counter
+
+import android.widget.Toast
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+
+class CounterViewModel : ViewModel() {
+
+    private val defaultCount = 0
+
+    private val _counter = MutableLiveData<Int>()
```

Comment:
```suggestion
    private val _counter = MutableLiveData<Int>()
    val counter: LiveData<Int>
        get() = _counter
```
가급적이면 private setter public getter는 붙여주시는 것이 좋습니다.